packages/storybook-utils/README.md

```diff
@@ -0,0 +1,15 @@
+<p>
+  <a href="https://gruppe.schwarz">
+  <div align="center">
+    <img src="https://raw.githubusercontent.com/SchwarzIT/onyx/main/.github/schwarz-group.svg" width="400px" />
+    </div>
+  </a>
+</p>
+
+# Onyx Storybook utils
+
+[Storybook](https://storybook.js.org) utilities for Vue created by [Schwarz IT](https://it.schwarz).
```

Comment:
do you want to add this kind of text to the `packages/figma-utils/README.md` aswell? (...created by schwarz it)